tests/test_glean_checks.py

```diff
@@ -0,0 +1,124 @@
+import pytest
+
+from probe_scraper.glean_checks import check_for_duplicate_metrics
+from probe_scraper.parsers.repositories import Repository
+
+OLD_DATE = "2019-10-04 17:30:42"
+NEWER_DATE = "2019-12-04 17:30:42"
+NEWEST_DATE = "2020-12-04 17:30:42"
+
+BASE_METADATA = {"notification_emails": ["foo@bar.com"]}
+
+
+@pytest.fixture
+def fake_repositories():
+    return [
+        Repository("glean-core", dict(BASE_METADATA, library_names=["glean-core"])),
+        Repository(
+            "glean-android", dict(BASE_METADATA, library_names=["glean-android"])
+        ),
+        Repository(
+            "fake-app",
+            dict(BASE_METADATA, dependencies=["glean-core", "glean-android"]),
+        ),
+    ]
+
+
+def test_check_duplicate_metrics_no_duplicates(fake_repositories):
+    # no overlap between metrics defined by glean-core and glean-android (both used by burnham)
```

Comment:
Oooh, so this was the issue? burnham and the others depends on `glean-deprecated`, which contains both the core metrics and the android ones?

Reply:
This comment is a little forward looking 😄 . #285 removes glean-deprecated and just adds glean-android explicitly to burnham and friends for backwards compat

Reply:
Ah, got it. But still, the mistake was on us for not separating these two things out .... 2 years ago.

Reply:
Yep, though it's hard to see the implications of doing things the quick way in the beginning. Happy we're in a better place now, at least.